src/app/course/course.module.ts

```diff
@@ -26,8 +28,10 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
     SharedModule,
     FormsModule,
     ReactiveFormsModule,
+    MatChipsModule,
   ],
 
   exports: [CourseListComponent, CourseItemComponent, AddCourseContainerComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],

```

Comment:
That's not clear why you add the custom schema. Is it because of Material components?